elasticsearch/kibana/map.jinja

```diff
@@ -1,7 +1,12 @@
 {% set kibana = salt['grains.filter_by']({
     'default': {
+        'pkgs': [
+            'kibana',
+            'nginx',
+        ],
+        'nginx_site_path': '/etc/nginx/sites-enabled',
         'config': {
-            'elasticsearch_url': 'http://localhost:9200',
+            'elasticsearch.url': 'http://localhost:9200',
```

Comment:
Will need to add a condition here as it changed in 5.x:
https://www.elastic.co/guide/en/kibana/current/settings.html